src/vscode/superbol-vscode-platform/superbol_cfg_explorer.mli

```diff
@@ -0,0 +1,19 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                        SuperBOL OSS Studio                             *)
+(*                                                                        *)
+(*                                                                        *)
+(*  Copyright (c) 2023 OCamlPro SAS                                       *)
+(*                                                                        *)
+(*  All rights reserved.                                                  *)
+(*  This source code is licensed under the MIT license found in the       *)
+(*  LICENSE.md file in the root directory of this source tree.            *)
+(*                                                                        *)
+(*                                                                        *)
+(**************************************************************************)
+
+val open_cfg
+  : ?typ: [< `Arc | `Dot > `Dot]
```

Comment:
For clarity of this interface I suggest to simply declare `type cfg_type = Graphviz | D3_arc_diagram` (adjust those constructors if they don't correspond).